--- postfix-3.0.0/conf/post-install.0002~	2015-03-17 09:39:41.654497126 +0100
+++ postfix-3.0.0/conf/post-install	2015-03-17 09:40:52.390834117 +0100
@@ -526,6 +526,10 @@ test -n "$create" && {
 	    case $path in
 	    no|no/*) continue;;
 	    esac
+	    # Munge paths for alternatives.
+	    case $path in
+		/usr/sbin/sendmail) path=$path.postfix ;;
+	    esac
 	    # Pick up the flags.
 	    case $flags in *u*) upgrade_flag=1;; *) upgrade_flag=;; esac
 	    case $flags in *c*) create_flag=1;; *) create_flag=;; esac
