diff -up postfix-3.7.2/conf/master.cf.1~ postfix-3.7.2/conf/master.cf
--- postfix-3.7.2/conf/master.cf.1~	2021-06-12 16:25:57.000000000 +0200
+++ postfix-3.7.2/conf/master.cf	2022-06-20 20:18:28.568219526 +0200
@@ -5,6 +5,9 @@
 #
 # Do not forget to execute "postfix reload" after editing this file.
 #
+# The script postfix-chroot.sh can be used to set up a Postfix chroot
+# environment on your OpenMandriva Lx System.
+#
 # ==========================================================================
 # service type  private unpriv  chroot  wakeup  maxproc command + args
 #               (yes)   (yes)   (no)    (never) (100)
@@ -51,6 +54,8 @@ smtp      inet  n       -       n
 #  -o milter_macro_daemon_name=ORIGINATING
 #628       inet  n       -       n       -       -       qmqpd
 pickup    unix  n       -       n       60      1       pickup
+  -o content_filter=
+  -o receive_override_options=
 cleanup   unix  n       -       n       -       0       cleanup
 qmgr      unix  n       -       n       300     1       qmgr
 #qmgr     unix  n       -       n       300     1       oqmgr
@@ -91,7 +96,7 @@ postlog   unix-dgram n  -       n
 # Also specify in main.cf: maildrop_destination_recipient_limit=1
 #
 #maildrop  unix  -       n       n       -       -       pipe
-#  flags=DRXhu user=vmail argv=/usr/local/bin/maildrop -d ${recipient}
+#  flags=DRXhu user=nobody argv=/usr/bin/maildrop -d ${recipient}
 #
 # ====================================================================
 #
@@ -129,6 +134,7 @@ postlog   unix-dgram n  -       n
 # ====================================================================
 #
 # Other external delivery methods.
+# These are not distributed with OpenMandriva Lx
 #
 #ifmail    unix  -       n       n       -       -       pipe
 #  flags=F user=ftn argv=/usr/lib/ifmail/ifmail -r $nexthop ($recipient)
@@ -143,3 +149,54 @@ postlog   unix-dgram n  -       n
 #mailman   unix  -       n       n       -       -       pipe
 #  flags=FRX user=list argv=/usr/lib/mailman/bin/postfix-to-mailman.py
 #  ${nexthop} ${user}
+#
+##### START OF CONTENT FILTER CUSTOMIZATIONS #####
+# Please see the Postfix FILTER_README for details.
+# These sample entries expect your content filter to
+# listen on port 10025 and to inject mail back into
+# postfix on port 10026.
+#
+# to enable such content filter run the command
+#    postconf -e content_filter=smtp-filter:127.0.0.1:10025
+#    postconf -e smtp-filter_destination_concurrency_limit=2
+# or
+#    postconf -e content_filter=lmtp-filter:127.0.0.1:10025
+#    postconf -e lmtp-filter_destination_concurrency_limit=2
+# and the command
+#    postconf -e receive_override_options=no_address_mappings
+#
+# adjust the value of ?mtp-filter_destination_concurrency_limit
+# to match the maximum number of process your content filter
+# will spawn.
+#
+127.0.0.1:10026        inet    n       -       n       -       -       smtpd
+  -o content_filter=
+  -o smtpd_restriction_classes=
+  -o smtpd_client_restrictions=permit_mynetworks,reject
+  -o smtpd_helo_restrictions=
+  -o smtpd_sender_restrictions=
+  -o smtpd_end_of_data_restrictions=
+  -o smtpd_etrn_restrictions=
+  -o smtpd_data_restrictions=
+  -o smtpd_delay_reject=no
+  -o smtpd_recipient_restrictions=permit_mynetworks,reject
+  -o mynetworks=127.0.0.0/8
+  -o smtpd_authorized_xforward_hosts=127.0.0.0/8
+  -o strict_rfc821_envelopes=yes
+  -o smtpd_error_sleep_time=0
+  -o smtpd_soft_error_limit=1001
+  -o smtpd_hard_error_limit=1000
+  -o receive_override_options=no_unknown_recipient_checks,no_header_body_checks
+#
+lmtp-filter    unix    -      -        n       -       -       lmtp
+  -o lmtp_data_done_timeout=1200
+  -o lmtp_send_xforward_command=yes
+  -o max_use=20
+#  -o lmtp_cache_connection=no
+#
+smtp-filter    unix    -      -        n       -       -       smtp
+  -o smtp_data_done_timeout=1200
+  -o smtp_send_xforward_command=yes
+  -o max_use=20
+#
+##### END OF CONTENT FILTER CUSTOMIZATIONS #####
diff -up postfix-3.7.2/conf/postfix-files.1~ postfix-3.7.2/conf/postfix-files
--- postfix-3.7.2/conf/postfix-files.1~	2022-01-21 19:50:18.000000000 +0100
+++ postfix-3.7.2/conf/postfix-files	2022-06-20 20:18:28.568219526 +0200
@@ -141,8 +141,6 @@ $command_directory/postqueue:f:root:$set
 $sendmail_path:f:root:-:755
 $newaliases_path:l:$sendmail_path
 $mailq_path:l:$sendmail_path
-$config_directory/LICENSE:f:root:-:644:1
-$config_directory/TLS_LICENSE:f:root:-:644:1
 $config_directory/access:f:root:-:644:p1
 $config_directory/aliases:f:root:-:644:p1
 $config_directory/bounce.cf.default:f:root:-:644:1
@@ -153,6 +151,7 @@ $config_directory/generics:f:root:-:644:
 $config_directory/header_checks:f:root:-:644:p1
 $config_directory/install.cf:f:root:-:644:o
 $config_directory/main.cf.default:f:root:-:644:1
+$config_directory/main.cf.dist:f:root:-:644:1
 $config_directory/main.cf:f:root:-:644:p
 $config_directory/master.cf:f:root:-:644:p
 $config_directory/pcre_table:f:root:-:644:o
@@ -254,6 +253,7 @@ $sample_directory/sample-misc.cf:f:root:
 $sample_directory/sample-pcre-access.cf:f:root:-:644:o
 $sample_directory/sample-pcre-body.cf:f:root:-:644:o
 $sample_directory/sample-pcre-header.cf:f:root:-:644:o
+$sample_directory/sample-pgsql.cf:f:root:-:644:o
 $sample_directory/sample-pgsql-aliases.cf:f:root:-:644:o
 $sample_directory/sample-qmqpd.cf:f:root:-:644:o
 $sample_directory/sample-rate.cf:f:root:-:644:o
@@ -314,7 +314,6 @@ $readme_directory/POSTSCREEN_3_5_README:
 $readme_directory/POSTSCREEN_README:f:root:-:644
 $readme_directory/QMQP_README:f:root:-:644:o
 $readme_directory/QSHAPE_README:f:root:-:644
-$readme_directory/RELEASE_NOTES:f:root:-:644
 $readme_directory/RESTRICTION_CLASS_README:f:root:-:644
 $readme_directory/SASL_README:f:root:-:644
 $readme_directory/SCHEDULER_README:f:root:-:644
diff -up postfix-3.7.2/HISTORY.1~ postfix-3.7.2/HISTORY
--- postfix-3.7.2/HISTORY.1~	2022-04-28 01:36:44.000000000 +0200
+++ postfix-3.7.2/HISTORY	2022-06-20 20:18:28.571552883 +0200
@@ -6945,7 +6945,7 @@ Apologies for any names omitted.
 	This involved a further rewrite of the regexp map code.
 	File: util/dict_regexp.c.
 
-200209010
+20020910
 
 	Bugfix: the SMTP client produced suprious warnings about
 	trouble with fallback_relay hosts. File:  smtp/smtp_connect.c.
@@ -6953,7 +6953,7 @@ Apologies for any names omitted.
 	Robustness: don't wait with detecting broken SMTP connections
 	until reading input. Leandro Santi. File: smtpd/smtpd_chat.c.
 
-200209011
+20020911
 
 	Workaround: IRIX 6 can't do ioctl FIONREAD on pipes. This
 	breaks the in_flow_delay feature.  File:  util/sys_defs.h.
diff -up postfix-3.7.2/proto/INSTALL.html.1~ postfix-3.7.2/proto/INSTALL.html
--- postfix-3.7.2/proto/INSTALL.html.1~	2021-12-20 23:11:11.000000000 +0100
+++ postfix-3.7.2/proto/INSTALL.html	2022-06-20 20:18:28.571552883 +0200
@@ -163,6 +163,7 @@ Linux Slackware 3.x and later <br>
 Linux SuSE 5.x and later <br>
 Linux Ubuntu 4.10 and later<br>
 Mac OS X <br>
+OpenMandriva Lx (any version)<br>
 NEXTSTEP 3.x <br>
 NetBSD 1.x and later <br>
 OPENSTEP 4.x <br>
diff -up postfix-3.7.2/proto/LDAP_README.html.1~ postfix-3.7.2/proto/LDAP_README.html
--- postfix-3.7.2/proto/LDAP_README.html.1~	2021-12-20 23:11:11.000000000 +0100
+++ postfix-3.7.2/proto/LDAP_README.html	2022-06-20 20:18:28.571552883 +0200
@@ -63,9 +63,9 @@ package.  </p>
 <p> Note 1: Postfix no longer supports the LDAP version 1 interface.
 </p>
 
-<p> Note 2: to use LDAP with Debian GNU/Linux's Postfix, all you
-need is to install the postfix-ldap package and you're done.  There
-is no need to recompile Postfix. </p>
+<p> Note 2: to use LDAP with OpenMandriva Lx or Debian GNU/Linux's
+Postfix, all you need is to install the postfix-ldap package and
+you're done.  There is no need to recompile Postfix. </p>
 
 <p> You need to have LDAP libraries and include files installed
 somewhere on your system, and you need to configure the Postfix
diff -up postfix-3.7.2/proto/MYSQL_README.html.1~ postfix-3.7.2/proto/MYSQL_README.html
--- postfix-3.7.2/proto/MYSQL_README.html.1~	2021-12-20 23:11:11.000000000 +0100
+++ postfix-3.7.2/proto/MYSQL_README.html	2022-06-20 20:18:28.571552883 +0200
@@ -38,9 +38,9 @@ code as described in the INSTALL documen
 be required if you build Postfix from a vendor-specific source
 package.  </p>
 
-<p> Note: to use mysql with Debian GNU/Linux's Postfix, all you
-need is to install the postfix-mysql package and you're done.
-There is no need to recompile Postfix. </p>
+<p> Note: to use mysql with OpenMandriva Lx or Debian GNU/Linux's
+Postfix, all you need is to install the postfix-mysql package and
+you're done.  There is no need to recompile Postfix. </p>
 
 <p> The Postfix MySQL client utilizes the mysql client library,
 which can be obtained from: </p>
diff -up postfix-3.7.2/proto/PCRE_README.html.1~ postfix-3.7.2/proto/PCRE_README.html
--- postfix-3.7.2/proto/PCRE_README.html.1~	2022-06-20 20:18:28.571552883 +0200
+++ postfix-3.7.2/proto/PCRE_README.html	2022-06-20 20:20:06.408931869 +0200
@@ -32,8 +32,7 @@ itself can be found at http://www.pcre.o
 
 <h2>Using Postfix packages with PCRE support</h2>
 
-<p> To use pcre with Debian GNU/Linux's Postfix, or with Fedora or
-RHEL Postfix, all you
+<p> To use pcre with OpenMandriva Lx's Postfix, all you
 need is to install the postfix-pcre package and you're done.  There
 is no need to recompile Postfix. </p>
 
diff -up postfix-3.7.2/proto/PGSQL_README.html.1~ postfix-3.7.2/proto/PGSQL_README.html
--- postfix-3.7.2/proto/PGSQL_README.html.1~	2021-04-22 01:05:28.000000000 +0200
+++ postfix-3.7.2/proto/PGSQL_README.html	2022-06-20 20:18:28.571552883 +0200
@@ -38,9 +38,9 @@ code as described in the INSTALL documen
 be required if you build Postfix from a vendor-specific source
 package.  </p>
 
-<p> Note: to use pgsql with Debian GNU/Linux's Postfix, all you
-need to do is to install the postfix-pgsql package and you're done.
-There is no need to recompile Postfix. </p>
+<p> Note: to use pgsql with OpenMandriva Lx or Debian GNU/Linux's
+Postfix, all you need to do is to install the postfix-pgsql package
+and you're done.  There is no need to recompile Postfix. </p>
 
 <p> In order to build Postfix with pgsql map support, you specify
 -DHAS_PGSQL, the directory with the PostgreSQL header files, and
diff -up postfix-3.7.2/README_FILES/INSTALL.1~ postfix-3.7.2/README_FILES/INSTALL
--- postfix-3.7.2/README_FILES/INSTALL.1~	2021-12-20 23:12:03.000000000 +0100
+++ postfix-3.7.2/README_FILES/INSTALL	2022-06-20 20:18:28.571552883 +0200
@@ -157,7 +157,7 @@ Randomization):
 (Specify "make makefiles pie=no" to explicitly disable Postfix position-
 independent executable support).
 
-Postfix PIE support appears to work on Fedora Core 20, Ubuntu 14.04, FreeBSD 9
+Postfix PIE support appears to work on Fedora Core 20, Ubuntu 14.04, OpenMandriva Lx, FreeBSD 9
 and 10, and NetBSD 6 (all with the default system compilers).
 
 Whether the "pie=yes" above has any effect depends on the compiler. Some
diff -up postfix-3.7.2/README_FILES/LDAP_README.1~ postfix-3.7.2/README_FILES/LDAP_README
--- postfix-3.7.2/README_FILES/LDAP_README.1~	2021-12-20 23:12:03.000000000 +0100
+++ postfix-3.7.2/README_FILES/LDAP_README	2022-06-20 20:18:28.571552883 +0200
@@ -32,9 +32,9 @@ from a vendor-specific source package.
 
 Note 1: Postfix no longer supports the LDAP version 1 interface.
 
-Note 2: to use LDAP with Debian GNU/Linux's Postfix, all you need is to install
-the postfix-ldap package and you're done. There is no need to recompile
-Postfix.
+Note 2: to use LDAP with OpenMandriva Lx or Debian GNU/Linux's Postfix, all you
+need is to install the postfix-ldap package and you're done. There is no need
+to recompile Postfix.
 
 You need to have LDAP libraries and include files installed somewhere on your
 system, and you need to configure the Postfix Makefiles accordingly.
diff -up postfix-3.7.2/README_FILES/MYSQL_README.1~ postfix-3.7.2/README_FILES/MYSQL_README
--- postfix-3.7.2/README_FILES/MYSQL_README.1~	2021-12-20 23:12:03.000000000 +0100
+++ postfix-3.7.2/README_FILES/MYSQL_README	2022-06-20 20:18:28.571552883 +0200
@@ -21,9 +21,9 @@ These instructions assume that you build
 in the INSTALL document. Some modification may be required if you build Postfix
 from a vendor-specific source package.
 
-Note: to use mysql with Debian GNU/Linux's Postfix, all you need is to install
-the postfix-mysql package and you're done. There is no need to recompile
-Postfix.
+Note: to use mysql with OpenMandriva Lx or Debian GNU/Linux's Postfix, all you
+need is to install the postfix-mysql package and you're done. There is no need
+to recompile Postfix.
 
 The Postfix MySQL client utilizes the mysql client library, which can be
 obtained from:
diff -up postfix-3.7.2/README_FILES/PCRE_README.1~ postfix-3.7.2/README_FILES/PCRE_README
--- postfix-3.7.2/README_FILES/PCRE_README.1~	2022-06-20 20:18:28.571552883 +0200
+++ postfix-3.7.2/README_FILES/PCRE_README	2022-06-20 20:20:41.005848379 +0200
@@ -16,9 +16,8 @@ pcre_table(5) manual page. Information a
 
 UUssiinngg PPoossttffiixx ppaacckkaaggeess wwiitthh PPCCRREE ssuuppppoorrtt
 
-To use pcre with Debian GNU/Linux's Postfix, or with Fedora or RHEL Postfix,
-all you need is to install the postfix-pcre package and you're done. There is
-no need to recompile Postfix.
+To use pcre with OpenMandriva's Postfix, all you need is to install the
+postfix-pcre package and you're done. There is no need to recompile Postfix.
 
 BBuuiillddiinngg PPoossttffiixx ffrroomm ssoouurrccee wwiitthh PPCCRREE ssuuppppoorrtt
 
diff -up postfix-3.7.2/README_FILES/PGSQL_README.1~ postfix-3.7.2/README_FILES/PGSQL_README
--- postfix-3.7.2/README_FILES/PGSQL_README.1~	2021-04-23 00:08:54.000000000 +0200
+++ postfix-3.7.2/README_FILES/PGSQL_README	2022-06-20 20:18:28.571552883 +0200
@@ -21,9 +21,9 @@ These instructions assume that you build
 in the INSTALL document. Some modification may be required if you build Postfix
 from a vendor-specific source package.
 
-Note: to use pgsql with Debian GNU/Linux's Postfix, all you need to do is to
-install the postfix-pgsql package and you're done. There is no need to
-recompile Postfix.
+Note: to use pgsql with OpenMandriva Lx or Debian GNU/Linux's Postfix, all you
+need to do is to install the postfix-pgsql package and you're done. There is no
+need to recompile Postfix.
 
 In order to build Postfix with pgsql map support, you specify -DHAS_PGSQL, the
 directory with the PostgreSQL header files, and the location of the libpq
diff -up postfix-3.7.2/src/global/mail_params.h.1~ postfix-3.7.2/src/global/mail_params.h
--- postfix-3.7.2/src/global/mail_params.h.1~	2022-04-09 00:10:07.000000000 +0200
+++ postfix-3.7.2/src/global/mail_params.h	2022-06-20 20:18:28.574886241 +0200
@@ -548,11 +548,7 @@ extern bool var_percent_hack;
   * Local delivery: alias databases.
   */
 #define VAR_ALIAS_MAPS		"alias_maps"
-#ifdef HAS_NIS
-#define DEF_ALIAS_MAPS		ALIAS_DB_MAP ", nis:mail.aliases"
-#else
 #define DEF_ALIAS_MAPS		ALIAS_DB_MAP
-#endif
 extern char *var_alias_maps;
 
  /*
diff -up postfix-3.7.2/src/util/sys_defs.h.1~ postfix-3.7.2/src/util/sys_defs.h
--- postfix-3.7.2/src/util/sys_defs.h.1~	2021-12-05 19:59:27.000000000 +0100
+++ postfix-3.7.2/src/util/sys_defs.h	2022-06-20 20:18:28.574886241 +0200
@@ -61,7 +61,7 @@
 #define ALIAS_DB_MAP   DEF_DB_TYPE ":/etc/mail/aliases"	/* OpenBSD 2.7 */
 #endif
 #ifndef ALIAS_DB_MAP
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #endif
 #define GETTIMEOFDAY(t)	gettimeofday(t,(struct timezone *) 0)
 #define ROOT_PATH	"/bin:/usr/bin:/sbin:/usr/sbin"
@@ -235,7 +235,7 @@
 #define HAS_DB
 #define HAS_SA_LEN
 #define NATIVE_DB_TYPE	"hash"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #define GETTIMEOFDAY(t) gettimeofday(t,(struct timezone *) 0)
 #define ROOT_PATH	"/bin:/usr/bin:/sbin:/usr/sbin"
 #define USE_STATFS
@@ -292,11 +292,11 @@
 /* might be set by makedef */
 #ifdef HAS_DB
 #define NATIVE_DB_TYPE	"hash"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #else
 #define HAS_DBM
 #define	NATIVE_DB_TYPE	"dbm"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #endif
 extern int optind;
 extern char *optarg;
@@ -764,7 +764,7 @@ extern int initgroups(const char *, int)
 #define HAS_FSYNC
 #define HAS_DB
 #define NATIVE_DB_TYPE	"hash"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #ifndef NO_NIS
 #define HAS_NIS
 #endif
@@ -847,7 +847,7 @@ extern int initgroups(const char *, int)
 #define HAS_FSYNC
 #define HAS_DB
 #define NATIVE_DB_TYPE	"hash"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #ifndef NO_NIS
 #define HAS_NIS
 #endif
@@ -880,7 +880,7 @@ extern int initgroups(const char *, int)
 #define HAS_FSYNC
 #define HAS_DB
 #define NATIVE_DB_TYPE	"hash"
-#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/aliases"
+#define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/postfix/aliases"
 #ifndef NO_NIS
 #define HAS_NIS
 #endif
