diff -up postfix-3.7.3/makedefs.omv~ postfix-3.7.3/makedefs
--- postfix-3.7.3/makedefs.omv~	2022-10-14 16:56:10.390504528 +0200
+++ postfix-3.7.3/makedefs	2022-10-14 16:56:28.385635378 +0200
@@ -627,7 +627,7 @@ EOF
 		: ${SHLIB_ENV="LD_LIBRARY_PATH=`pwd`/lib"}
 		: ${PLUGIN_LD="${CC-gcc} -shared"}
 		;;
- Linux.[345].*)	SYSTYPE=LINUX$RELEASE_MAJOR
+ Linux.[3456789].*)	SYSTYPE=LINUX$RELEASE_MAJOR
 		case "$CCARGS" in
 		 *-DNO_DB*) ;;
 		 *-DHAS_DB*) ;;
diff -up postfix-3.7.3/src/util/sys_defs.h.omv~ postfix-3.7.3/src/util/sys_defs.h
--- postfix-3.7.3/src/util/sys_defs.h.omv~	2022-10-14 16:58:43.763614663 +0200
+++ postfix-3.7.3/src/util/sys_defs.h	2022-10-14 16:58:59.821730288 +0200
@@ -751,7 +751,7 @@ extern int initgroups(const char *, int)
  /*
   * LINUX.
   */
-#if defined(LINUX2) || defined(LINUX3) || defined(LINUX4) || defined(LINUX5)
+#if defined(LINUX2) || defined(LINUX3) || defined(LINUX4) || defined(LINUX5) || defined(LINUX6) || defined(LINUX7) || defined(LINUX8) || defined(LINUX9)
 #define SUPPORTED
 #define UINT32_TYPE	unsigned int
 #define UINT16_TYPE	unsigned short
