--- postfix-2.10.2/src/util/dict_db.c.orig	2014-03-17 16:02:00.549526406 +0000
+++ postfix-2.10.2/src/util/dict_db.c	2014-03-17 16:07:55.827537764 +0000
@@ -699,7 +699,7 @@
 	if ((errno = db->upgrade(db,db_path,0)) != 0)
 	    msg_fatal("upgrade of database %s: %m",db_path);
     }
-#if DB_VERSION_MAJOR == 5 || (DB_VERSION_MAJOR == 4 && DB_VERSION_MINOR > 0)
+#if DB_VERSION_MAJOR == 6 || DB_VERSION_MAJOR == 5 || (DB_VERSION_MAJOR == 4 && DB_VERSION_MINOR > 0)
     if ((errno = db->open(db, 0, db_path, 0, type, db_flags, 0644)) != 0)
 	FREE_RETURN(dict_surrogate(class, path, open_flags, dict_flags,
 				   "open database %s: %m", db_path));
