diff -up postfix-3.4.2/makedefs.omv~ postfix-3.4.2/makedefs
--- postfix-3.4.2/makedefs.omv~	2019-03-11 17:30:26.104060553 +0100
+++ postfix-3.4.2/makedefs	2019-03-11 17:30:37.135242719 +0100
@@ -557,7 +557,7 @@ EOF
 		: ${SHLIB_ENV="LD_LIBRARY_PATH=`pwd`/lib"}
 		: ${PLUGIN_LD="${CC-gcc} -shared"}
 		;;
-  Linux.[34].*)	SYSTYPE=LINUX$RELEASE_MAJOR
+  Linux.[3-9].*) SYSTYPE=LINUX$RELEASE_MAJOR
 		case "$CCARGS" in
 		 *-DNO_DB*) ;;
 		 *-DHAS_DB*) ;;
