--- postfix-3.1.2/conf/master.cf.omv~	2016-09-18 19:32:49.856624113 +0200
+++ postfix-3.1.2/conf/master.cf	2016-09-18 22:44:57.234850904 +0200
@@ -84,6 +84,9 @@ scache	unix	-	-	y	-	1	scache
 #  flags=DRhu user=nobody argv=/usr/bin/maildrop -d ${recipient}
 #
 # ====================================================================
+# Dovecot delivery
+dovecot		unix	-	n	n	-	-	pipe
+  flags=DRhu user=vmail:mail argv=/usr/libexec/dovecot/deliver -f ${sender} -d ${recipient}
 #
 # Please See the Postfix CYRUS_README file for details
 # deliver interface (deprecated), to use this also use
