diff -up postfix-3.5.0/conf/master.cf.6~ postfix-3.5.0/conf/master.cf
--- postfix-3.5.0/conf/master.cf.6~	2020-04-03 00:25:25.948657720 +0200
+++ postfix-3.5.0/conf/master.cf	2020-04-03 00:26:08.752859781 +0200
@@ -87,6 +87,10 @@ postlog   unix-dgram n  -       n
 #  flags=DRXhu user=nobody argv=/usr/bin/maildrop -d ${recipient}
 #
 # ====================================================================
+# Dovecot delivery
+dovecot                unix    -       n       n       -       -       pipe
+  flags=DRhu user=vmail:mail argv=/usr/libexec/dovecot/deliver -f ${sender} -d ${recipient}
+# ====================================================================
 #
 # Recent Cyrus versions can use the existing "lmtp" master.cf entry.
 #
